### PR TITLE
fix(links): Fix links for some experiment installation

### LINKED ISSUE #341: Pre-requisite link for K8 Experiment is showing 404
STR

https://docs.litmuschaos.io/docs/Kubernetes-Chaostoolkit-AWS/

In this section, the pre-requisite link is not working.
Link given is https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/chaostoolkit/k8-aws-ec2-terminate/experiment.yaml

It has to be modified to 
https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/generic/k8-pod-delete/experiment.yaml

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>

#### Issue:
- fixes: https://github.com/litmuschaos/litmus-docs/issues/341

#### Details:

- This PR is fixing some links for installing chaos experiments in the pre-requisite section.  The experiments are:
  - aws-ec2-terminate
  - k8s-pod-delete
  - k8s-cluster-kiam
  - k8s-service-kill

### CODE REVIEW

File: docs/chaostoolkit-k8-service-delete.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ sidebar_label: Application Service
 
 ## Prerequisites
 - Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
-- Ensure that the `k8-pod-delete` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace. If not, install from [here](kubectl apply -f https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/chaostoolkit/k8-pod-delete/experiment.yaml)
+- Ensure that the `k8-pod-delete` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace. If not, install from [here](https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/generic/k8-pod-delete/experiment.yaml)
```
Comment (ajeshbaby) [inline docs/chaostoolkit-k8-service-delete.md]: Link is https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/k8-service-kill/experiment.yaml

**Review by ajeshbaby — DISMISSED** [review]
LGTM

File: docs/chaostoolkit-k8-cluster-kiam.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ sidebar_label: Cluster Pod - kiam
 
 ## Prerequisites
 - Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
-- Ensure that the `k8-pod-delete` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace. If not, install from [here](kubectl apply -f https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/chaostoolkit/k8-pod-delete/experiment.yaml)
+- Ensure that the `k8-pod-delete` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace. If not, install from [here](https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/generic/k8-pod-delete/experiment.yaml)
```
Comment (ajeshbaby) [inline docs/chaostoolkit-k8-cluster-kiam.md:25]: https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/k8-pod-delete/experiment.yaml

File: docs/chaostoolkit-k8-pod-delete.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ sidebar_label: Service Pod - Application
 
 ## Prerequisites
 - Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
-- Ensure that the `k8-pod-delete` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace. If not, install from [here](kubectl apply -f https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/chaostoolkit/k8-pod-delete/experiment.yaml)
+- Ensure that the `k8-pod-delete` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace. If not, install from [here](https://hub.litmuschaos.io/api/chaos/1.7.0?file=charts/generic/k8-pod-delete/experiment.yaml)
```
Comment (ajeshbaby) [inline docs/chaostoolkit-k8-pod-delete.md:25]: https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/k8-pod-delete/experiment.yaml

File: docs/chaostoolkit-k8-cluster-kiam.md (hunk 2 above)
Comment (uditgaurav) [reply docs/chaostoolkit-k8-cluster-kiam.md:25]: @ajeshbaby We keep the latest version in the master branch.

**Review by ajeshbaby — APPROVED** [review]
LGTM

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**